.gitlab-ci.yml

```diff
@@ -15,11 +15,7 @@
     - ${PWD}/tests/test.sh
 
     # Make sure OpenWISP is running.
-    - >
-      docker exec ${container_id} curl --insecure -s --head https://localhost/admin/login/?next=/admin/
-      | sed -n '1p' | grep -q '200'
-      && (echo 'Status code 200 test: pass' && exit 0)
-      || (echo 'Status code 200 test: fail'; docker exec ${container_id} sh -c 'tail -n 500 /opt/openwisp2/log/*.log' && exit 1)
+    - ./scripts/run-openwisp.sh
```

Comment:
`./scripts/run-openwisp.sh` is not helpful as a name and as a path. I'd simply use `./tests/runtests.sh` as it's at least similar with the `runtests.py` and `runtests` scripts we have in other repos.

Reply:
HI, +1 to the comment, sorry have been busy lately :)